scripts/install_from_release.sh

```diff
@@ -1,24 +1,29 @@
 # Script to install scenarios file.  Pass in language code parameter for a particular language, such as it-it for Italian.
 set -e
 
-# Define the language parameter
+# TODO: make parameters mandatory
 LANG="$1"
```

Comment:
Could we add a step to our primary testing pipeline which makes sure that the installation script always works and does a few things? The two things I have in mind for the test are to check if:

1. Installs IE on to the PATH
2. Can access the scenarios folder